src/settings-ui/Settings.UI.Library/MousePointerCrosshairsProperties.cs

```diff
@@ -31,7 +31,7 @@ public class MousePointerCrosshairsProperties
 
         public MousePointerCrosshairsProperties()
         {
-            ActivationShortcut = new HotkeySettings(false, true, true, false, 0x50); // Ctrl + Alt + P
+            ActivationShortcut = new HotkeySettings(true, false, true, false, 0x50); // Win + Alt + P
```

Comment:
from a pure nerd / DRY approach, i hate how the default values are not centralized.